Source/Csla.AspNetCore/Blazor/ApplicationContextManagerBlazor.cs

```diff
@@ -194,8 +204,10 @@ public IContextDictionary GetClientContext(ApplicationContext.ExecutionLocations
     /// </summary>
     /// <param name="clientContext">Client context.</param>
     /// <param name="executionLocation"></param>
+    /// <exception cref="InvalidOperationException"><see cref="ApplicationContext"/> is <see langword="null"/>.</exception>
     public void SetClientContext(IContextDictionary clientContext, ApplicationContext.ExecutionLocations executionLocation)
     {
+      ThrowIfApplicationContextIsNull();
```

Comment:
null is valid here

Reply:
Exception comes from `ApplicationContext` could be `null`.